### ISSUE TITLE
Glassware

### ISSUE THREAD

**Issue by jaxbot** [BarcodeEye/BarcodeEye#12]
Is the BarcodeEye team actively planning to make this official Glassware? If not, I think someone should. Very useful piece of software that should be easily available to all users.

**Comment by jromero** [comment]
See https://github.com/BarcodeEye/BarcodeEye/issues/6 it was on hold until the GDK resolved some issues. I have not tested it with the latest GDK to see what the user experience is like now but if it's improved the goal would be to make it Glassware.

**Comment by jaxbot** [comment]
Ah, I see. The experience is fairly decent when I tried it out last week on 16.11. The trigger needs to be changed to something like "RECOGNIZE_THIS" or a request needs to be put in for "SCAN_BARCODE", but otherwise it seems like it could make it in after some rounds of polish. I'm willing to contribute to make this happen if needed, as well.

**Comment by jromero** [comment]
The project is always open to contribution, including polish. Originally SCAN_BARCODE was rejected (as mentioned on #6 ) I might try again since I didn't see them add a SCAN_PRODUCT in the commands. If you've got time can you make a pull request for changing to a RECOGNIZE_THIS voice trigger since that is the most suitable at the moment.

**Comment by jaxbot** [comment]
Got an email back from Glassware team after I requested "scan this barcode". They seem pretty set on "recognize this" being used instead

**Comment by jromero** [comment]
Good to know. Thank you for the update.

**Comment by jaxbot** [comment]
So what's the status on this? The app is functional on the latest XE version and recognize this can be used; what else needs to be done before trying get Glass Team to approve it?

**Comment by paoloburzacca** [comment]
Hi guys, I compiled and deployed this app but i cannot see it in the list of glass application, i have the last glass release XE21

**Comment by jaxbot** [comment]
I'm not sure if BarcodeEye has been updated to support the latest XE versions. I have a fork at https://github.com/jaxbot/glass-qrlens that should work though, and it's precompiled. Tell me if it does; if not, more XE changes are needed. I don't have XE21 yet

**Comment by jaxbot** [comment]
After several rounds of the Glass review team sending feedback, getting the UI up to spec, etc., my port "QR Lens" is now live in My Glass: https://glass.google.com/glassware/15728809402747867797
